mne/io/brainvision/brainvision.py

```diff
@@ -64,12 +69,14 @@ class RawBrainVision(_BaseRaw):
     @verbose
     def __init__(self, vhdr_fname, montage=None,
                  eog=('HEOGL', 'HEOGR', 'VEOGb'), misc=(),
-                 scale=1., preload=False, response_trig_shift=0, verbose=None):
+                 scale=1., preload=False, response_trig_shift=0,
+                 event_id={}, verbose=None):
```

Comment:
we try to avoid mutable defaults so let's go with `None` here. Or is there a difference between `dict()` and `None`?